content/en/docs/tutorials/cluster-management/install-use-dra.md

```diff
@@ -0,0 +1,500 @@
+---
+title: Install and Use DRA Drivers
+content_type: tutorial
+weight: 60
+min-kubernetes-server-version: v1.32
+---
+
+{{< feature-state feature_gate_name="DynamicResourceAllocation" >}}
+
+
+<!-- overview -->
+This page contains a high level background on the third-party and Kubernetes
+native components involved in providing the Dynamic Resource Allocation (DRA)
+feature, in the context of cluster administration and operations. This page then
+explains what to expect when installing DRA drivers in your cluster and how to
+use them in conjunction with the DRA APIs to request and observe the allocation
+of a Pod's hardware claim.
+
+<!-- objectives -->
+
+### {{% heading "objectives" %}}
+
+* Install an example DRA driver
+* Deploy a Pod requesting a hardware claim using DRA APIs
+* Remove the Pod and observe the cleanup
+
+<!-- prerequisites -->
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+Your cluster also must be configured to use the Dynamic Resource Allocation
+feature. DRA APIs are stable as of Kubernetes version 1.34 but can be disabled.
```

Comment:
TODO: not sure exactly where stable vs on by default landed in 1.34, update this line.

Also, should I just copy over the #enable-dra directions from the other page and put them inline? The other page goes on to do a lot of other stuff I don't really want the tutorial user to do, but also don't want to copy pasta too much.